platform.md

```diff
@@ -293,16 +293,16 @@ Usage:
 ```
 
 ##### Inputs
-| Input         | Environment Variable    | Default Value             | Description
-|---------------|-------------------------|---------------------------|----------------------
-| `<app>`         | `CNB_APP_DIR`         | `/workspace`          | Path to application directory
-| `<buildpacks>`  | `CNB_BUILDPACKS_DIR`  | `/cnb/buildpacks`     | Path to buildpacks directory (see [Buildpacks Directory Layout](#buildpacks-directory-layout))
-| `<group>`       | `CNB_GROUP_PATH`      | `<layers>/group.toml` | Path to output group definition
-| `<layers>`      | `CNB_LAYERS_DIR`      | `/layers`             | Path to layers directory
-| `<log-level>`   | `CNB_LOG_LEVEL`       | `info`                | Log Level
-| `<order>`       | `CNB_ORDER_PATH`      | `/cnb/order.toml`     | Path to order definition (see [`order.toml`](#ordertoml-toml))
-| `<plan>`        | `CNB_PLAN_PATH`       | `<layers>/plan.toml`  | Path to output resolved build plan
-| `<platform>`    | `CNB_PLATFORM_DIR`    | `/platform`           | Path to platform directory
+| Input         | Environment Variable    | Default Value                            | Description
+|---------------|-------------------------|------------------------------------------|-------
+| `<app>`         | `CNB_APP_DIR`         | `/workspace`                             | Path to application directory
+| `<buildpacks>`  | `CNB_BUILDPACKS_DIR`  | `/cnb/buildpacks`                        | Path to buildpacks directory (see [Buildpacks Directory Layout](#buildpacks-directory-layout))
+| `<group>`       | `CNB_GROUP_PATH`      | `<layers>/group.toml`                    | Path to output group definition
+| `<layers>`      | `CNB_LAYERS_DIR`      | `/layers`                                | Path to layers directory
+| `<log-level>`   | `CNB_LOG_LEVEL`       | `info`                                   | Log Level
+| `<order>`       | `CNB_ORDER_PATH`      | `<layers>/order.toml`, `/cnb/order.toml` | Path to order definition (see [`order.toml`](#ordertoml-toml))
```

Comment:
if we doing ordering for `CNB_ORDER_PATH` should we follow the PATH with a separator like `:`:

```suggestion
| `<order>`       | `CNB_ORDER_PATH`      | `<layers>/order.toml`:`/cnb/order.toml` | Path to order definition (see [`order.toml`](#ordertoml-toml))
```

Reply:
To me the `:` notation makes it seem like you can set `CNB_ORDER_PATH` to a path list. Are we actually proposing that?

Or, should we use more words to make it clear. E.g. either "`<layer>/order.toml` if present, or `/cnb/order.toml`" or a section at the top that states that when multiple `,` separated defaults are specified they are listed in order of decreasing precedence.

Reply:
I liked your suggestion of wording @ekcasey and updated it.